<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenRGBSettingsPageUi</class>
 <widget class="QWidget" name="OpenRGBSettingsPageUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="11" column="1">
    <widget class="QCheckBox" name="CheckboxAutoStartSetServerPort">
     <property name="text">
      <string>Set Server Port</string>
     </property>
    </widget>
   </item>
   <item row="18" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="ThemeLabel">
     <property name="text">
      <string>Theme</string>
     </property>
    </widget>
   </item>
   <item row="15" column="6">
    <widget class="QComboBox" name="ComboBoxAutoStartProfile"/>
   </item>
   <item row="0" column="6">
    <widget class="QComboBox" name="ComboBoxTheme"/>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="CheckboxMinimizeOnClose">
     <property name="text">
      <string>Minimize On Close</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QCheckBox" name="CheckboxAutoStartServer">
     <property name="text">
      <string>Start Server</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QCheckBox" name="CheckboxAutoStart">
     <property name="text">
      <string>Start At Login</string>
     </property>
    </widget>
   </item>
   <item row="11" column="6">
    <widget class="QLineEdit" name="TextServerPort">
     <property name="inputMask">
      <string>90000</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="QCheckBox" name="CheckboxAutoStartProfile">
     <property name="text">
      <string>Load Profile</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QCheckBox" name="CheckboxAutoStartClient">
     <property name="text">
      <string>Start Client</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QCheckBox" name="CheckboxAutoStartMinimized">
     <property name="text">
      <string>Start Minimized</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="CheckboxSaveGeometry">
     <property name="text">
      <string>Save Geometry On Close</string>
     </property>
    </widget>
   </item>
   <item row="17" column="1">
    <widget class="QLabel" name="AutoStartStatusLabel">
     <property name="text">
      <string>Start at Login Status</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLabel" name="AutoStartLabel">
     <property name="text">
      <string>Start At Login Settings:</string>
     </property>
    </widget>
   </item>
   <item row="13" column="6">
    <widget class="QLineEdit" name="TextClientHost"/>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="CheckboxLoadGeometry">
     <property name="text">
      <string>Load Window Geometry</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QCheckBox" name="CheckboxAutoStartCustom">
     <property name="text">
      <string>Custom Arguments</string>
     </property>
    </widget>
   </item>
   <item row="14" column="6">
    <widget class="QLineEdit" name="TextCustomArgs"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
